managed-services/data-lab/how-to/create-data-lab.mdx

```diff
@@ -0,0 +1,39 @@
+---
+meta:
+  title: How to create a Distributed Data Lab
+  description: Step-by-step guide to creating a Distributed Data Lab on Scaleway.
+content:
+  h1: How to create a Distributed Data Lab
+  paragraph: Step-by-step guide to creating a Distributed Data Lab on Scaleway.
+tags: distributed data lab apache spark create process
+dates:
+  validation: 2024-07-31
+  posted: 2024-07-31
+categories:
+  - managed-services
+  - data-lab
+---
+
+Distributed Data Lab is a product designed to assist data scientists and data engineers in performing calculations on a remotely managed Apache Spark infrastructure.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- [Signed up to the private beta](https://www.scaleway.com/fr/betas/#distributed-data-lab) and received a confirmation email.
+- An [Object Storage bucket](/storage/object/how-to/create-a-bucket/)
+- A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
+
+1. Click **Data Lab** under **Managed Services** on the side menu. The Distributed Data Lab page displays.
+
+2. Click **Create Data Lab cluster**. The creation wizard displays.
+
+3. Complete the following steps in the wizard:
```

Comment:
suggestion: notebook version selection is missing

Reply:
Only one option is supported, will update procedure, when other options will be available